### DISCUSSION TITLE
How to release Linux system programs?

### DISCUSSION THREAD

**Discussion by Jason0247** [airsdk/Adobe-Runtime-Support#3580] [Q&A]
Hello everyone, I can say that I know nothing about publishing Linux system programs.
May I ask if it is not possible to use Animate when publishing Linux programs?
Do I need to purchase authorization documents in order to test, and do I also need to purchase authorization? If necessary, how much does the authorization cost for one year?
thank you.

**Comment by pol2095** [comment]
Hello,
it's possible to generate the swf using Animate on Windows or macOS and after using `adt` command line to generate a captive runtime on Ubuntu x86_64 or ARM64 or other linux system (it's necessary to install OpenJDK on Ubuntu).
You need too a [commercial Harman license](https://airsdk.harman.com/pricing).

**Reply by Jason0247** [reply]
Is it impossible for me to test without a license?

**Reply by pol2095** [reply]
No, you need a license to test on Linux.
